.jenkins.yml

```diff
@@ -3,3 +3,9 @@ pull_requests:
 
 pushes:
   - main
+
+notifications:
+  - name: Build failures on the main branch
+    branch: master
```

Comment:
@mst-ableton @nre-ableton this should read
```yaml
    branch: main
```

Reply:
See #6